src/synthesizer/sed.py

```diff
@@ -965,11 +966,9 @@ def get_photo_luminosities(self, filters, verbose=True):
             # and the spectral grid are the same array
             if not np.array_equal(f.lam, self.lam):
                 if verbose:
```

Comment:
should this verbose flag still be here?

Reply:
Absolutely not! It's been obliterated with maximum prejudice.